src/ncp/ncp_base.cpp

```diff
@@ -342,7 +342,7 @@ void NcpBase::HandleActiveScanResult(otActiveScanResult *result)
             SPINEL_PROTOCOL_TYPE_THREAD,
             flags,
             result->mNetworkName,
-            result->mExtPanId, sizeof(result->mExtPanId)
+            result->mExtPanId, Thread::Mac::Beacon::kExtPanIdSize
```

Comment:
Well in this case, I think, using `OT_EXT_PAN_ID_SIZE` define would be better than `sizeof(otExtAddress)`, because extended address and extended PAN ID are not intrinsically the same thing, and might be a little bit misleading. Another way could be to add typedef/struct otExtPanId to openthread-types.h and use sizeof of that.